src/data/list.js

```diff
@@ -0,0 +1,176 @@
+const mongoCollections = require('../config/mongoCollections');
+const boardData = require('./boards');
+const boards = mongoCollections.boards;
+const {ObjectId} = require('mongodb');
+const error_handler = require('../errors/error-handler'); 
+
+
+module.exports = {
+    /**
+    * Gets all lists from a given board id.
+    * @param {String} id The board's id.
+    * @returns A list of list(column) objects.
+    */
+    getAllListsInBoard: async (id) => {
+        if(!id || !error_handler.checkObjectId(id)) 
+            throw new Error("id is not valid.");
+
+        const board = await boardData.readById(id);
+        let returnList = [];
+        for(let x=0; x<board.lists.length; x++) {
+            let lists = board.lists[x];
+            lists._id = lists._id.toString();
+            returnList.push(lists);
+        }
+
+        return returnList;
+    },
+
+    /**
+    * Gets a list with a given id.
+    * @param {String} listId The lists's id.
+    * @param {String} boardId The board's id.
```

Comment:
Why do you need boardId here? Security purposes?

Reply:
Security and otherwise we would have to check every board we have on our servers, but we should only call this function on a given board so we would have the id

Reply:
Fair enough